docs/api/machine-learning/sync.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[machine-learning-api-sync]]
+=== Sync {ml} saved objects API
+++++
+<titleabbrev>Sync {ml} saved objects</titleabbrev>
+++++
+
+Synchronizes {kib} saved objects for {ml} jobs.
+
+[[machine-learning-api-sync-request]]
+==== Request
+
+`GET <kibana host>:<port>/api/ml/saved_objects/sync`
+
+`GET <kibana host>:<port>/s/<space_id>/api/ml/saved_objects/sync`
+
+
+[[machine-learning-api-sync-path-params]]
+==== Path parameters
+
+`space_id`::
+(Optional, string) An identifier for the space. If `space_id` is not provided in
+the URL the default space is used.
+
+[[machine-learning-api-sync-query-params]]
+==== Query parameters
+
+`simulate`::
+(Optional, boolean) When `true`, simulates the synchronization by only returning
+the list actions that _would_ be performed.
+
+[[machine-learning-api-sync-response-body]]
+==== Response body
```

Comment:
it might be good to mention that these returned objects contain lists of job ids which have been synced and a success flag.

Reply:
Good idea!  I've augmented those descriptions in ec5c928c2506228e711d5f318291c8973bb551e4